crates/subspace-farmer-components/src/plotting.rs

```diff
@@ -248,11 +248,13 @@ where
     (PieceOffset::ZERO..)
         .zip(raw_sector.records.iter_mut())
         .zip(sector_contents_map.iter_record_bitfields_mut())
-        // TODO: Doesn't work without a bridge: https://github.com/ferrilab/bitvec/issues/143
-        .par_bridge()
+        // TODO: Ideally, we'd use parallelism here, but using `.par_bridge()` causes Chia table
+        //  derivation to only use a single thread, which slows everything to essentially
```

Comment:
Do you know why this happens?

Reply:
When we use `par_bridge()` or anything else `rayon` will literally run one instance of a function per thread. Hence when you try to run something rayon-based in there it'll only have that one thread available (they are all running on a shared thread pool). While we could create a separate thread pool, it would mean you can't control it globally or from the outside.

I think we'll be exploring this more in the future because right now parallelism doesn't quite use all the cores on my machine. But we can't run any number of things in parallel either because each table uses over 600MiB, so by running many of them in parallel we'll quickly run out of memory on most machines.

For now what I have in this PR is an okay compromise, but I'm open for suggestions on how we can do better.